### ISSUE TITLE
Logo tweaking

### ISSUE THREAD

**Issue by cherrymathew** [decant/decant#1]
The logo is that of a funnel, which is used in a different way from decanting.

Decanting is normally done from a vessel, by exploiting relative densities.

The logo needs to reflect that, IMHO.

**Comment by fraggerfox** [comment]
> Decanting is normally done from a vessel, by exploiting relative densities.

Correct, we need the logo to contain a decanting funnel or a variation of a decanting funnel.
